spec/views/admin/offers/show.html.erb_spec.rb

```diff
@@ -130,13 +130,45 @@
 
       it 'allows you to mark the offer as lapsed' do
         click_link('Offer Lapsed')
+        expect(page).to have_content('Undo lapse')
         expect(page).to have_content("Offer ##{offer.reference_id}")
         expect(page).to have_content('State lapsed')
         expect(page).to_not have_selector('.offer-draft-actions')
         expect(page).to_not have_selector('.offer-actions')
       end
     end
 
+    describe 'undo lapse' do
+      before do
+        offer.update!(state: 'review')
+        stub_gravity_artist(id: submission.artist_id)
+        page.visit "/admin/offers/#{offer.id}"
+      end
+
+      it 'shows the undo lapse button' do
+        expect(page).to have_content('Accept Offer')
+        expect(page).to_not have_selector('.offer-draft-actions')
+        expect(page).to have_selector('.offer-actions')
```

Comment:
**(nitpick, non-blocking):** might be good to have an explicit check for 'Undo Lapse' button here rather than just parent div

Reply:
Ok. I will change it